packages/ui/src/components/search-files.tsx

```diff
@@ -61,39 +64,36 @@ export const SearchFiles = ({ navigateToFile, filesList }: SearchFilesProps) =>
     <Popover open={isOpen} onOpenChange={setIsOpen}>
       <PopoverAnchor asChild>
         <div>
-          <SearchBox.Root
-            width="full"
-            placeholder="Search files..."
-            className="searchbox h-9"
-            handleChange={handleInputChange}
-            value={query}
-          />
+          <SearchBox.Root width="full" placeholder="Search files..." handleChange={handleInputChange} value={query} />
         </div>
       </PopoverAnchor>
       <PopoverContent
-        className="max-h-60 w-[600px] overflow-auto p-0"
+        className="w-[300px] p-0"
         align="start"
         onOpenAutoFocus={event => {
           event.preventDefault()
         }}
       >
         <Command>
-          <CommandList>
+          <CommandList heightClassName="max-h-60">
             <CommandEmpty>No file found.</CommandEmpty>
-            <CommandGroup>
-              {filteredFiles?.map((file: string, idx: number) => (
-                <CommandItem
-                  key={idx}
-                  value={file}
-                  onSelect={() => {
-                    navigateToFile(file)
-                    setIsOpen(false)
-                  }}
-                >
-                  {fileText(file)}
-                </CommandItem>
-              ))}
-            </CommandGroup>
+            {filteredFiles && !!filteredFiles.length && (
+              <CommandGroup>
+                {filteredFiles?.map((file: string, idx: number) => (
```

Comment:
`filteredFiles` will always exist and be an array, so you can simplify
```suggestion
            {!!filteredFiles.length && (
              <CommandGroup>
                {filteredFiles.map((file: string, idx: number) => (
```